python/ray/serve/handle.py

```diff
@@ -181,6 +136,7 @@ def _options(
         method_name: Union[str, DEFAULT] = DEFAULT.VALUE,
         multiplexed_model_id: Union[str, DEFAULT] = DEFAULT.VALUE,
         stream: Union[bool, DEFAULT] = DEFAULT.VALUE,
+        use_new_handle_api: Union[bool, DEFAULT] = DEFAULT.VALUE,
```

Comment:
`DEFAULT.VALUE` actually _is_ used (subtly). Consider this case:
`handle.options(use_new_handle_api=True).options(stream=True)`

If the default for `use_new_handle_api` were `False` instead of `DEFAULT.VALUE`, the second `.options` would overwrite it.